### ISSUE TITLE
Libgiles1-mesa-dev has no installation candidate

### ISSUE THREAD

**Issue by KrishnaRajputPK** [openframeworks/openFrameworks#6360]
[](url)

**Comment by kronikle** [comment]
same here with debian 10.0.0

**Comment by kr15h** [comment]
Same on 
```
Linux raspberrypi 4.19.57-v7+ #1244 SMP Thu Jul 4 18:45:25 BST 2019 armv7l GNU/Linux
```

**Comment by Tenkir** [comment]
Same here on `Raspbian GNU/Linux 10 (buster)`

**Comment by Tenkir** [comment]
Looks like Buster removed support for `libgiles1`

**Comment by Strum** [comment]
Same Debian 10

**Comment by ofTheo** [comment]
For #6438 getting OF to work on Buster for the Rpi it just involved switching the install dependencies from `libgiles1-mesa-dev` to use `libgles1` 

See https://github.com/openframeworks/openFrameworks/pull/6438/files#diff-7263e131fbc295fdf3622c0c204a498aR40 

Imagine it might be just a simple on desktop.

**Comment by Strum** [comment]
Thanks @ofTheo , that sorted it out, building now.

**Comment by Strum** [comment]
Maybe a note could be added to the install instructions for relevant systems?

**Comment by ofTheo** [comment]
Is it as simple as switching the install_dependcies.sh to `libgles1` all will that break Stretch and older?

**Comment by ofTheo** [comment]
Closed by #6438
